lua/cfc_prop_restoration/core/cl/cl_init.lua

```diff
@@ -0,0 +1,6 @@
+
+net.Receive( "Restore_AlertReconnectingPlayer", function( len )
+    -- CREATE NOTIFICATION --
+    net.Start( "Restore_RestorePlayerProps" )
+    net.SendToServer()
+end)
```

Comment:
I think you're supposed to have a space, correct me if I'm wrong though

Reply:
good catch, ill fix it in a bit

Reply:
That wasn't very descriptive, I think you need to add a space between `end` and `)`